easybuild/easyblocks/o/openfoam.py

```diff
@@ -181,6 +181,8 @@ def configure_step(self):
 
         # inject compiler variables into wmake/rules files
         ldirs = glob.glob(os.path.join(self.builddir, self.openfoamdir, 'wmake', 'rules', 'linux*'))
+        if self.version.startswith('v') and LooseVersion(self.version) >= LooseVersion('v1906'):
```

Comment:
@mboisson please sync with current `develop` and use `self.looseversion > LooseVersion('1906'):`, see changes in #1794

Reply:
Done